src/Sylius/Bundle/CoreBundle/Doctrine/ORM/ShippingMethodRepository.php

```
@@ -62,7 +93,11 @@
             ->andWhere(':channel MEMBER OF o.channels')
             ->setParameter('channel', $channel)
             ->setParameter('enabled', true)
-            ->addOrderBy('o.position', 'ASC')
-        ;
+            ->addOrderBy('o.position', 'ASC');
+    }
+
+    private function buildChannelConfigurationPattern(string $channelCode): string
+    {
+        return sprintf('%%"%s":%%', $channelCode);
     }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'buildChannelConfigurationPattern'. (undefined)

(UnusedPrivateMethod)